examples/contour/01-contour-config.yaml

```diff
@@ -95,6 +95,7 @@ data:
     #   - "upstream_service_time"
     #   - "user_agent"
     #   - "x_forwarded_for"
+    #   - "grpc_status"
```

Comment:
in order for `grpc_status` to be a default field logged with JSON logging, you will have to add to this list: https://github.com/projectcontour/contour/blob/667dc3ff0f0b9ad545bb9746be8232b968c22e8c/pkg/config/accesslog.go#L17

Reply:
also will require an addition to the testing here and the accompanying implementation: https://github.com/projectcontour/contour/blob/667dc3ff0f0b9ad545bb9746be8232b968c22e8c/pkg/config/parameters_test.go#L194

otherwise actually using that field will not succeed, it will be rejected as invalid

Reply:
similarly for https://github.com/projectcontour/contour/blob/667dc3ff0f0b9ad545bb9746be8232b968c22e8c/pkg/config/parameters_test.go#L506

Reply:
Done. Added the test cases and to the default list.